RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -183,5 +222,13 @@ private string GetNextTestModuleName(IVBProject project)
 
             return string.Concat(TestModuleBaseName, index);
         }
+
+        private IEnumerable<Declaration> GetDeclarationsToStub(Declaration parentDeclaration)
+        {
+            return _state.AllUserDeclarations
```

Comment:
You could use `_state.DeclarationFinder.Members(parentDeclaration);` instead, to get the module's members without iterating anything - the finder uses an encapsulated dictionary to pull a module's members.